conf.d/teamcity.yaml.example

```diff
@@ -22,6 +22,9 @@ instances:
     # rather than just that a successful build happened
     # is_deployment: true
 
+    # Optional, this turns of ssl ceritificate validation. Defaults to True.
```

Comment:
Small typos here: `turns of ssl ceritificate validation` -> `turns off ssl certificate validation`